LA 100% RPS files

ssattler opened the pull request:
These are the state files needed to model the LA 100% state RPS: the out of state fraction table (oosfrac), rec trading definition (rectable), and RPS definition by year (rps_fraction).

Definitions from Paul Arbaje on 5/12/25 in the state RPS review.xls file
https://ucsusa.sharepoint.com/:x:/s/PowerAheadTeam/EXUunZguG6lDjJAwlCWr7DgB0rVhrM8cKPV1ygQJsNVQoA?e=sBVH11&clickparams=eyAiWC1BcHBOYW1lIiA6ICJNaWNyb3NvZnQgT3V0bG9vayIsICJYLUFwcFZlcnNpb24iIDogIjE2LjAuMTg3MzAuMjAxNDIiLCAiT1MiIDogIldpbmRvd3MiIH0%3D&CID=0E6B5EC3-9DEB-4CA5-B22D-AD99599E3176&wdLOR=c70C6F8F8-DD18-4DBB-96AD-E3A93BB603EE

ssattler committed:
LA 100% RPS files

These are the state files needed to model the LA 100% state RPS: the out of state fraction table (oosfrac), rec trading definition (rectable), and RPS definition by year (rps_fraction).

samgdotson commented on inputs/state_policies/rectable.csv:
```diff
@@ -1,38 +1,61 @@
+NB,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,1,1,0,0
+NS,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,0,0,0
+voluntary,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,0
+,,,,,,,,,,,,,,,,,,,,,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```
Are these rows necessary? They seem to be missing information or erroneous… the original file seems to be “square” and there are now a bunch of rows with singular zero values. Was this intended?

ssattler replied:
Thanks for catching. When I changed the rectable with the new LA numbers, that pushed the previous zeros down. I'll upload the correct file soon.

samgdotson commented on inputs/state_policies/rps_fraction.csv:
```diff
@@ -1,1307 +1,1348 @@
+2034,WI,0.098920841,0,0
+2035,WI,0.098770891,0,0
+2036,WI,0.098759187,0,0
+2037,WI,0.098822676,0,0
```
I don’t understand why the wisconsin RPS would fluctuate around 10%… is this correct?

ssattler replied:
That is from NREL. I only changed the LA numbers for this run.

samgdotson reviewed (changes requested):
I don’t see any obvious formatting errors. But I do have a couple of questions regarding the files that should be answered before merging. 

Thanks!

ssattler committed:
revised rectable to remove additional zeros

I removed the zeros from the rectable file (the values were pushed down when I copied Paul's LA trading values into this file).

samgdotson reviewed (approved):
Looks good, thanks for making the changes!
